jdaviz/components/plugin_auto_label.vue

```diff
@@ -4,7 +4,7 @@
       <v-text-field
         ref="textField"
         :value="displayValue"
-        @keyup="if(auto) {$emit('update:auto', false)}; $emit('update:value', $event.srcElement._value)"
+        @keyup="if(auto) {if ($event.srcElement._value === displayValue) {return}; $emit('update:auto', false)}; $emit('update:value', $event.srcElement._value)"
```

Comment:
this basically says to only disable to auto behavior and update the traitlet if the keypress has resulted in a widget value (`srcElement._value`) that is different from the one in the traitlet(s) (`displayValue`).... effectively triggering the `return` without doing anything for other keypresses.